src/coach/GetBallCoach.cpp

```diff
@@ -18,7 +18,7 @@ bool GetBallCoach::shouldWeGetBall(const Field &field) {
     // should probably listen to ballPossession at some point
     Vector2 ballPos = world_new::World::instance()->getWorld()->getBall()->get()->getPos();
     return !FieldComputations::pointIsInDefenceArea(field, ballPos, true, 0.04) && !FieldComputations::pointIsInDefenceArea(field, ballPos, false) &&
-           FieldComputations::pointIsInField(field, ballPos, -0.05);
+           FieldComputations::pointIsInField(field, ballPos, 0.05);
```

Comment:
Why did you change the -?

Reply:
Margin from now on are all outwards. Some margins in the FieldComputations where inwards instead of outwards.